src/riak_kv_sweeper_fold.erl

```diff
@@ -465,10 +465,14 @@ update_stat_counters(NumMutated, NumDeleted, RObjBinSize, Acc) ->
 -spec stat_send(riak_kv_sweeper:index(),
                 {'final'| 'in_progress', #sa{}}) -> #sa{}.
 stat_send(Index, {final, Acc}) ->
```

Comment:
Just a note, list comprehensions [do not generate a list](http://erlang.org/doc/efficiency_guide/listHandling.html#id68285), when the compiler can detect that the return value is not used.